parsers/IL.py

```diff
@@ -55,14 +55,32 @@ def flatten_list(_2d_list) -> list:
     return flatten_list(cleaned_list)
 
 
+def fetch_price(zone_key="IL", session=None, target_datetime=None, logger=None) -> dict:
+    """Fetch price from IEC table."""
+    if target_datetime is not None:
+        raise NotImplementedError('This parser is not yet able to parse past dates')
+
+    response = get(IEC_PRICE)
+    soup = BeautifulSoup(response.content, "lxml")
+
+    price = soup.find("td", class_="ms-rteTableEvenCol-6")
```

Comment:
Can't we find a more stable selector than this? This is very fragile. As mentioned in the other PR, it would be great if we could stabilize the selectors.

Reply:
This is actually a very good selector, but as in any web scraping apllications its as stable as the site itself..
I am using it in my HomeAssistant as my source of price in the past years and it hasn't been changed

Reply:
I've looked into the page, and I see that we cannot do much unfortunately.

There are 54 elements on the page with the class `ms-rteTableEvenCol-6`. Are we just interested in the first one?

Reply:
Well, it depends..

The first one is what most people will pay (e.g. households, agricultural)
however - in israel the tariff is decided by both the consumer - if he wants a load based or fixed, and also by type (e.g. factories vs housholds)

I have picked first one as it seems the most logical choice for me. you can however vote against :)